sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -161,11 +212,27 @@ def callback(raw_response):
     @distributed_trace
     def list_submitted_jobs(self, **kwargs):
         # type: (**Any) -> ItemPaged[JobStatusResult]
-        """
-
-        :keyword int results_per_page:
-        :keyword int skip:
-        :rtype: ~azure.core.polling.ItemPaged[JobStatusResult]
+        """List all the submitted translation jobs under the Document Translation resource.
+
+        :keyword int top: Use top to indicate the total number of results you want
```

Comment:
my understanding was you guys weren't going to expose `top` and `skip` for this release, I could be wrong though